### PR TITLE
Functional tests - Add test 'Filter stores'

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add test 'Filter stores'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/13_shopParameters/05_contact/stores/*" URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20559)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParameters_contact_stores_filterStores';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+
```
Comment (NeOMakinG) [inline tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js]: There are some empty lines that shouldn't be there :P

File: tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+// Browser and tab
+let browserContext;
+let page;
+
```
Comment (NeOMakinG) [inline tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js:28]: same

File: tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+let numberOfStores = 0;
+
```
Comment (NeOMakinG) [inline tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js:30]: same

**Review by NeOMakinG — CHANGES_REQUESTED** [review]

File: tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js]: all fixed
thanks

**Review by NeOMakinG — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Pull request merged**